packages/gatsby-theme/src/components/Card/CardRichText.js

```diff
@@ -9,14 +9,14 @@ const CardRichText = ({ text, children, sx }) => (
 )
 
 CardRichText.propTypes = {
-  children: PropTypes.string,
+  children: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
I think this we could leave this is `PropTypes.node`. I think this will cover all the cases, but we can leave it as is.

Reply:
Thanks @dinartecc I'm going to change it 🙏